### DISCUSSION TITLE
Question on Binder Design

### DISCUSSION THREAD

**Discussion by jamesrgraham** [RosettaCommons/RFdiffusion#207] [Q&A]
Hello,

I was testing making a binder to a "random" PDB structure.

The protein is 209aa in length and I chose the hotspot residues and the region within the protein where those residues appear (67-209).

`'contigmap.contigs=[A67-209/0 70-100]' 'ppi.hotspot_res=[A68,A79,A183,A195]'`

The output contains two chains. 'A' is the poly-gly binder and 'B' appears to be the region from the target protein.

Is the second chain (the target region) simply included for the sake of convenience?

I manually extracted the 'A' chain to add sequence with ProteinMPNN and then made AF structures and am using that to test docking to the original PDB structure.

Is that a reasonable approach?

Thanks!
james

**Comment by roccomoretti** [comment]
Yes, that sounds somewhat reasonable. 

However, if you are interested in designing a binder, you may want to include the target protein during ProteinMPNN -- the sorts of residues you get at protein monomer surface are different than the types of residues you get at a protein-protein interface. In particular, hydrophobic pockets and hydrogen bonding groups on the target protein surface can change the probabilities of nearby residues. (For example by causing ProteinMPNN to use hydrophobic residues to insert into hydrophobic pockets on the target, or put hydrogen bonding residues near possible hbonding partners.)

**Comment by atumbio** [comment]
When you get binder design pdb files, how do you go about visualizing the connections to the molecule in question like what you have presented in the paper?  What softwares are you guys using?
